packages/@o3r/rules-engine/src/components/rules-engine/ruleset-history/ruleset-history-pres.template.html

```diff
@@ -11,8 +11,10 @@ <h4 class="mb-4">Ruleset Execution History</h4>
         [class.error]="execution.type === 'RulesetExecutionError'"
         (click)="toggleExpansion(execution.executionId, 'ruleset')">
         <div><span [title]="'This ruleset has been evaluated ' + execution.executionCounter + ' time(s)'">{{execution.executionCounter}}</span> - {{execution.rulesetName | titlecase }}
-          <div class="ruleset-panel-subtitle" *ngIf="execution.rulesetInformation?.linkedComponent as linkedComponent">
-            {{linkedComponent.name}} {{linkedComponent.library}}
+          <div class="ruleset-panel-subtitle" *ngIf="execution.rulesetInformation?.linkedComponents.or || execution.rulesetInformation?.linkedComponent">
+            <ng-container *ngFor="let lc of (execution.rulesetInformation?.linkedComponents.or || []).concat(execution.rulesetInformation?.linkedComponent) | unique; last as isLast">
```

Comment:
```suggestion
            <ng-container *ngFor="let lc of (execution.rulesetInformation.linkedComponents.or || [execution.rulesetInformation.linkedComponent]) | unique; last as isLast">
```

Reply:
with this change I do not support both linkedComponent and linkedComponents in the same time.

Reply:
let's support _linkedComponents_ first and fallback on _linkedComponent_.